app/assets/javascripts/helpers/APIRoutes.js

```diff
@@ -28,7 +28,10 @@ class ApiRoutes {
 
   get buyers() {
     return {
-      requests: id => `/api/buyers/requests/${id}`
+      index       : `/api/buyers`,
```

Comment:
I think we don't need an index route? I also think that this route isn't being used anywhere